request.go

```diff
@@ -494,7 +499,8 @@ func handleTime(attribute interface{}, args []string, fieldValue reflect.Value)
 func handleNumeric(
```

Comment:
It seems that we no longer support float IDs, is this function needed?

Reply:
Yes, attributes still can be of float type. Take a look at:  https://github.com/getoutreach/jsonapi/blob/49369807e985fdf44a9cc1f8a0cd7fe44862149a/request.go#L417